lib/wadler/print.rb

```diff
@@ -22,15 +23,20 @@ class Wadler
     # @param new_line [String]
     # @param out [Object] should have a write and string method
     # @param width [Integer]
+    # @param whitespace [String] the whitespace character. Used to trim trailing whitespaces.
+    #
+    # @see Token::Whitespace
```

Comment:
I think this `@see` should be attached to the `@param`.  So no empty line in between.